src/pages/EditMonitor.vue

```diff
@@ -532,6 +545,19 @@ export default {
         addedNotification(id) {
             this.monitor.notificationIDList[id] = true;
         },
+
+        /**
+         * Is the friendly name required for the selected option.
+         * Returns true if it is else false
+         * @returns {boolean}
+         */
+        friendlyNameRequired() {
+            if (this.friendlyNameRequiredOptions.includes(this.monitor.type)) {
+                return true;
+            } else {
+                return false;
+            }
```

Comment:
oh also here you can do just

`return this.friendlyNameRequiredOptions.includes(this.monitor.type);`

Reply:
Fixed. Thanks for your help 👍